parlai/core/torch_agent.py

```diff
@@ -123,8 +113,12 @@ def __init__(
         valid_indices=None,
         candidates=None,
         candidate_vecs=None,
+        reward=None,
```

Comment:
where is this `reward` coming from?

Reply:
Some teachers provide it now and we batchify it. It's used by Unlikelihood and others.

Reply:
could you include it in the `Batch` docstring?